carma_wm/include/carma_wm/SignalizedIntersectionManager.h

```diff
@@ -36,8 +36,9 @@ namespace carma_wm
 
 struct LANE_DIRECTION
 {
-  static const uint8_t INGRESS = 1;
-  static const uint8_t EGRESS = 2;
+  // Integration testing found reverse of j2735_msgs as below. TODO
```

Comment:
TODO mean anything here?

Reply:
I removed the TODO comment as there is nothing to do from carma-platform side, but I was thinking maybe Carma-cloud has a bug where it switched the types of lanes. 
From my understanding INGRESS is entry lane, while EGRESS is exit lane: 
https://github.com/usdot-fhwa-stol/carma-msgs/blob/develop/j2735_msgs/msg/LaneDirection.msg
But integration testing found it is sending the opposite

Reply:
I believe you have the correct interpretation? Can you follow up with Sudhakar about this? Might be an issue in v2x hub

Reply:
Apparently the ingress/egress is just a static file that can be edited. I have a general idea from Will on how to edit that XML file, so I reversed the INGRESS/EGRESS to match j2735